### PR TITLE
Rework GCP playbooks to use bootkube

### PR DESCRIPTION
**Pull request by vrutkovs**
Requires `openshift_gcp_bootstrap_ignition_file` - a location of bootstrap ignition file.

TODO:
* [x] Ensure multiple masters / workers are working
* [x] ~~Port the rest of the scripted gcloud commands to ansible~~
  These modules are community-supported and should not be actively used
* [x] Deprovision: remove DNS entries
* [x] Move playbooks to `test/gcp`
* [ ] Remove hacks, FIXMEs etc.
* [ ] Consider using google LB instead of DNS round-robin
* [ ] Run ignition tasks as async?

### CODE REVIEW

**Comment by michaelgugino** [comment]
@sdodson I think we should stick them in test/ and be explicit about for 'test use only'

**Comment by sdodson** [comment]
Decision from this morning's standup is that they'll live in openshift-ansible/test

File: roles/openshift_gcp/tasks/dynamic_inventory.yml [hunk 1]
```diff
@@ -1,5 +1,12 @@
 ---
 - name: Extract PEM from service account file
-  copy: content="{{ (lookup('file', openshift_gcp_iam_service_account_keyfile ) | from_json ).private_key }}" dest=/tmp/gce.pem mode=0600
+  copy:
+    content: "{{ (lookup('file', openshift_gcp_iam_service_account_keyfile ) | from_json ).private_key }}"
+    dest: /tmp/gce.pem
+    mode: 0600
```
Comment (patrickdillon) [inline roles/openshift_gcp/tasks/dynamic_inventory.yml]: mode: 0600 was overly strict when I ran this locally (uncontainerized). Modules trying to open the file would have permission denied.

Comment (michaelgugino) [reply roles/openshift_gcp/tasks/dynamic_inventory.yml]: @patrickdillon This is possibly due to incorrect use of 'become'.

**Pull request merged**